about.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: doc
+title: About the OBO Foundry
+---
+
+# About the OBO Foundry
+
+<p>The mission of the OBO Foundry is to develop a family of interoperable ontologies that are both logically well-formed and scientifically accurate.
+To achieve this, OBO Foundry participants follow and contribute to the development of an evolving set of <a href="/principles/fp-000-summary.html">principles</a>
+including <a href="/principles/fp-001-open.html">open use</a>, <a href="/principles/fp-010-collaboration.html">collaborative development</a>, <a href="/principles/fp-005-delineated-content.html">non-overlapping and strictly-scoped content</a>, and common <a href="/principles/fp-002-format.html">syntax</a> and <a href="/principles/fp-007-relations.html">relations</a>,
+based on ontology models that work well, such as the <a target="_new" href="http://geneontology.org/>Gene Ontology (GO)</a>.
+       
+<p>The OBO Foundry is overseen by an <a href="http://obofoundry.org/docs/OperationsCommittee.html">Operations Committee</a> with
+<a href="/docs/EditorialWG.html">Editorial</a>, <a href="/docs/TechnicalWG.html">Technical</a> and <a href="/docs/OutreachWG.html">Outreach</a> working groups.</a>
+      
+<p>You can contribute to the OBO Foundry by submitting issues or pull requests to our GitHub repo, <a href="https://github.com/OBOFoundry/OBOFoundry.github.io">OBOFoundry/OBOFoundry.github.io</a>,
```

Comment:
should we make a bulleted list here to try to avoid a WOT? :) (I love this acronym!)

Reply:
It's not a very long paragraph though, either way :)

Reply:
Let's see how it looks when it's on the web page. I suspect it will need some formatting tweaks.